users/README.md

```diff
@@ -111,6 +111,22 @@ Returns the specified user.
   - **Code:** 401
     - **Content:** `{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`
 
+## **GET /users/userAvailable/:id**
+
+Returns the availability of user name.
+
+- **Params**  
+  _Required:_ `id=[string]`
+- **Body**  
+  None
+- **Headers**  
+  Content-Type: application/json
+- **Cookie**  
+  rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:** `{ 'userAvailable': <Availability Stauts> }`
```

Comment:
Something like `isUsernameAvailable` sounds better to me. What do you say?
Also the value should be <boolean>